### PR TITLE
Feature/option clear jvmf

### LINKED ISSUE #354: Bug: JVMF breaks if JVMF array empty
As title says. 

JVMF has no check for no messages?

### PR DESCRIPTION
**Pull request by AdozenBABYdogs**
**When merged this pull request will:**
- Describe what this pull request will do
- Each change in a separate line
- Include documentation if applicable

This PR adds the ability to remove JVMF default messages via the addon settings

### CODE REVIEW

File: addons/uh60_jvmf/XEH_preStart.sqf [hunk 1]
```diff
@@ -1,3 +1,11 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+[
+	"vtx_uh60_jvmf_defaults"
+	,"CHECKBOX"
+	,["Enable JVMF Default Messages", "The JVMF Default messages provide an example of usage of the JVMF"]
+	,["UH-60M", "JVMF"]
+	[true];
```
Comment (jonpas) [inline addons/uh60_jvmf/XEH_preStart.sqf]: This doesn't look correct.

Comment (AdozenBABYdogs) [reply addons/uh60_jvmf/XEH_preStart.sqf]: Yep, I see it.  I'm missing a comma.  Does the syntax for cba_fnc_addSetting look correct?

Comment (AdozenBABYdogs) [reply addons/uh60_jvmf/XEH_preStart.sqf]: I also think the order on where I'm executing the code if off as well, and this function belongs in the "XEH_preInit.sqf"

Comment (jonpas) [reply addons/uh60_jvmf/XEH_preStart.sqf]: Bool, not an array.

Comment (AdozenBABYdogs) [reply addons/uh60_jvmf/XEH_preStart.sqf]: Thank you

**Comment by AdozenBABYdogs** [comment]
JVMF requires rewrite in order to support 0 messages in the table at the time of use

**Comment by AdozenBABYdogs** [comment]
Ready for merge

JVMF rewrite completed, JVMF hides when no messages are present.  JVMF Delete message button added to remove erroneous/non applicable JVMF messages.

**Review by Riverman02 — APPROVED** [review]
Let's try it

**Pull request merged**